lib/trackler/track.rb

```diff
@@ -119,21 +119,28 @@ def dir
     end
 
     def hints
-      if File.exist?(track_hints_filename)
-        File.read(track_hints_filename)
+      docfile = DocFile.find(basename: 'EXERCISE_README_INSERT', track_dir: dir, image_path: "")
+      # DocFile does not handle deprecations.
+      # Check if we found it, then fall back to
+      # deprecated files if we didn't find one.
+      return docfile.to_md unless docfile.name.empty?
```

Comment:
OK. 

The whole thing will go away once we get rid of the deprecated files anyway.